protobuf/datastore.proto

```diff
@@ -51,6 +51,11 @@ message Polygon { // horizontal area; three or more points
   repeated Point points = 1;
 }
 
+message Circle { // horizontal circle area
+  Point center = 1;
+  double radius = 2;
```

Comment:
Looking at the code this is in km, but this is not specified. I would have expected meters if not specified, but better to make it explicit.

Reply:
Do you prefer to have the unit in the variable name (`radius` -> `radius_km`), or does it suffice to add a comment like in this case:
```
message Point { // horizontal position
  double lat = 1; // latitude degrees in range [-90, 90]
  double lon = 2; // longitude degrees in range [-180, 180]
}
```
Either way I guess we should ensure consistency so that we follow the same practice everywhere?

Reply:
I guess with Issue #109 the unit won't be in the name at all since it is defined separately in `radius_unit`. So only documenting in the comment for now in order to minimize modifications required soon by implementation of #109.